ui/src/app/shared/service/utils.ts

```diff
@@ -659,13 +658,20 @@ export namespace HistoryUtils {
         return 0;
       }
     };
-
-    export const POSITIVE_AS_ZERO_AND_INVERT_NEGATIVE = (value) => {
+  
+export const POSITIVE_AS_ZERO_AND_INVERT_NEGATIVE = (value) => {

```

Comment:
Indentation is broken

Reply:
got some different vsc settings in my workspaces somehow